### PR TITLE
Adding phyx

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by druvus** [comment]
Holding merging this recipe until we have a prover version to refer to.

File: recipes/phyx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  url: https://github.com/FePhyFoFum/{{ name|lower }}/archive/84634ef7ea874cea98f88694ad2e0f95836164ac.tar.gz
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc
```
Comment (bgruening) [inline recipes/phyx/meta.yaml]: we should also use the new `compiler` here and the host section

Comment (druvus) [reply recipes/phyx/meta.yaml]: I will update once we have a new tagged release. I will remind them.

**Comment by druvus** [comment]
@bioconda-bot please update

**Comment by dpryan79** [comment]
@druvus Is this ready for review?

**Comment by druvus** [comment]
@dpryan79 Yep, I was going to close and open a new PR but you are welcome to review this one. I didn't think anyone would see this 2 year old PR.

**Comment by druvus** [comment]
@bioconda-bot please fetch artifacts

**Comment by druvus** [comment]
@bioconda-bot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
It was at the top of the PR list given how I have it sorted :)

Anyway, approved, so merge when ready.

**Comment by druvus** [comment]
@bioconda-bot please merge

**Pull request merged**